d2target/d2target.go

```diff
@@ -95,29 +95,24 @@ func (d *Diagram) getBoard(boardPath []string) *Diagram {
 		return d
 	}
 
+	if len(boardPath) < 2 {
+		return nil
```

Comment:
what about this other form? https://github.com/terrastruct/d2/blob/master/d2target/d2target.go#L76-L77
I think this won't find any boards since it is recursive and will hit this nil near the end

Reply:
yeah urgh, I can't in good conscience merge this without tests. will take a bit longer, shouldn't have put it up for review so prematurely